io/io/src/TStreamerInfo.cxx

```diff
@@ -2216,7 +2218,7 @@ void TStreamerInfo::BuildOld()
          // Case of a numerical type
          if (element->GetType() >= TStreamerInfo::kObject) {
             // Old type was not a numerical type.
-            element->SetNewType(-2);
+            element->SetNewType(kUnsupportedConversion);
```

Comment:
Why is this missing the `TVirtualStreamerInfo::`?

Reply:
Oversight :(